packages/clay-core/src/table/Cell.tsx

```diff
@@ -92,6 +97,7 @@ export const Cell = React.forwardRef<HTMLTableCellElement, Props>(
 			expanded,
 			index,
 			keyValue,
+			sortAriaLabel = 'Sort Column',
```

Comment:
We're building a standard to use a single API called `messages`, I'm going to move this there so I can just add it in one place.

https://github.com/liferay/clay/blob/c5675c0443b4ec25273a4172589e3656cdaabfd7/packages/clay-core/src/table/Table.tsx#L54